.travis.yml

```diff
@@ -22,8 +22,10 @@ env:
   matrix:
   - LINT_CHECK="1"
   - TRANSIFEX="1"
-  - TESTS="1" ODOO_REPO="odoo/odoo"
-  - TESTS="1" ODOO_REPO="OCA/OCB"
+  - TESTS="1" ODOO_REPO="odoo/odoo" INCLUDE=partner_multi_relation
```

Comment:
no clue, I didn't look into this version's code yet. But I wanted to give the rationale why it used to be tested separately